### ISSUE TITLE
Make goldeneye follow redirects

### ISSUE THREAD

**Issue by jseidl** [jseidl/GoldenEye#11]

**Comment by craig** [comment]
But make sure it only follows redirects to the same domain, otherwise an attacked webserver could reflect you elsewhere.

**Comment by jseidl** [comment]
Indeed. still thinking if I should add this feature or the end user should follow the redirect and use the final URL in the tool. Comments anyone?

**Comment by craig** [comment]
Some DDoS mitigations use redirects with tokens like this:
site.com/foo -> site.com/?token=23423421212
site.com/?token=23423421212 -> site.com/foo

So it would be nice if it followed redirects for the same domain only (or make that configurable).
I've been using the tool to test resilience of Anti-DDoS-Appliances against L7 DDoS attacks and I'm sure it would be helpful.
